bot.js

```diff
@@ -104,7 +104,7 @@ class SoundBot {
       return randomCommand + requestCommands;
     }).join('\n\n');
 
-    channel.sendMessage('Available sounds:\n```\n' + commandList + '```');
+    user.sendMessage('Available sounds:\n```\n' + commandList + '```').then(msg => message.delete(5000));
```

Comment:
`msg => message.delete(5000)` should be changed to `() => message.delete(5000)` so that the function doesn't have an unused parameter.